viper/modules/pehash/pehasher.py

```diff
@@ -20,18 +22,19 @@
 except ImportError:
     HAVE_BITSTRING = False
 
-from viper.common.out import *
+# from viper.common.out import *  # not used
+
 
 def calculate_pehash(file_path=None, data=None):
     if not HAVE_PEFILE:
+        # TODO (frennkie): what's going on here? should this be a Class (sub classing "Module")?
```

Comment:
You're right, this is going to fail... Need to check how this module is supposed to work.